roles/matrix-bot-maubot/templates/config/config.yaml.j2

```diff
@@ -3,10 +3,12 @@
 # Format examples:
 #   SQLite:   sqlite:///filename.db
 #   Postgres: postgresql://username:password@hostname/dbname
-database: sqlite:////data/maubot.db
+database: {{ matrix_bot_maubot_storage_database|to_json }}
 
 # Separate database URL for the crypto database. "default" means use the same database as above.
-crypto_database: default
+crypto_database: 
+    type: default
+    postgres_uri: {{ matrix_bot_maubot_storage_database|to_json }}
```

Comment:
Why do you introduce a seperate cryptodatabase here that uses postgres?

Reply:
According to the sample config this line is only used when the db is set to postgres.

Reply:
Ah okay. But is there a reason to specifically set it? In my understandig it would be enough to say that the crypto_database is the same as the "normal" database

Reply:
Not sure :shrug:

Reply:
Okay. I mean it doesn't hut so lets leave it for now